### DISCUSSION TITLE
Original Contracts Problem

### DISCUSSION THREAD

**Discussion by amirghiasi** [Aelto/W3_RandomEncounters_Tweaks#108] [Q&A] [answered]
Hey, I installed this mod and i really like it, thank you. but i have a problem, when i activate the contracts i cant get the mian contracts on boards anymore. it happend to me before i get ''Contract: Woodland Beast". when i disabled contracts on mod setting i was able to get that contract

**Comment by Aelto** [comment, marked as answer]
Hello,

This is a feature of the mod, you can read more about it in the last paragraph of [this page](https://aelto.github.io/tw3-random-encounters-reworked/rer-bible/contracts/index.html).

If you wish to disable the feature you can use the requirement slider in the contract system menu and set it to 0.

**Reply by amirghiasi** [reply]
Thanks again i going to read hole document now for setting
